client/packages/system/src/Asset/ListView/ListView.tsx

```diff
@@ -79,7 +80,7 @@ const AssetListComponent: FC = () => {
     },
   ];
 
-  if (isCentralServer) columnDescriptions.push('selection');
+  if (isCentralServer) columnDescriptions.unshift(GenericColumnKey.Selection);
```

Comment:
hehe one more 😅

Reply:
Ah the elusive one! Fixed here 808a5cb26c5f6e0847d99d6c64b27ea7bee6d4bc :)